functions/__sf_section_dir.fish

```diff
@@ -13,6 +13,8 @@ function __sf_section_dir -d "Display the current truncated directory"
 	__sf_util_set_default SPACEFISH_DIR_TRUNC 3
 	__sf_util_set_default SPACEFISH_DIR_TRUNC_REPO true
 	__sf_util_set_default SPACEFISH_DIR_COLOR cyan
+	__sf_util_set_default SPACEFISH_DIR_LOCK_SYMBOL "🔒"
+	__sf_util_set_default SPACEFISH_DIR_LOCK_COLOR red
```

Comment:
@matchai  @Snuggle  How do we set color to the SPACEFISH_DIR_LOCK_SYMBOL?
Because as in [spaceship](https://github.com/denysdovhan/spaceship-prompt/commit/a5d5d31aee54a00833b1f7114793290038d4e00f)  the ```zsh``` way is easier.
Our ```__sf_lib_section``` seems to be hindering this implementation.
Correct me if I'm wrong.

Reply:
@jskrnbindra Thanks for getting started on this PR! 😄 

Spaceship appears to circumvent the issue by having the lock symbol be displayed within the suffix using its color code, which I believe should work for spacefish as well.

Similarly to their implementation [here](https://github.com/denysdovhan/spaceship-prompt/commit/a5d5d31aee54a00833b1f7114793290038d4e00f), the following might work:

```fish
set SPACEFISH_DIR_SUFFIX (set_color $SPACEFISH_DIR_LOCK_COLOR)$SPACEFISH_DIR_LOCK_SYMBOL\
(set_color normal)$SPACESHIP_DIR_SUFFIX
```

Reply:
Cool. Worked nicely!